qa-contract/README.md

```
@@ -0,0 +1,246 @@
+# 4. Read back / verify indices (read-only; no key needed)
+node src/query.mjs                    # self-check: exercises every index
+node src/query.mjs --type testCase --tier Essential
+node src/query.mjs --type testRun --testId CORE-05 --proof
+```
+
+`--result` must be one of `pass | fail | blocked | skipped`. Add `--proof` to any
+`query.mjs` call to fetch with a verified Platform proof, `--json` for raw output.
+
+### Files
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)